os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -533,8 +503,9 @@ static int reloading_thread(int argc, char *argv[])
 	 * So the common binary should be unloaded after termination of all user binaries
 	 * to clear used resources normally.
 	 */
+	load_cmd = LOADCMD_LOAD_ALL;
 
-	/* 1. Unload all user binaries */
+	/* Unload all user binaries */
```

Comment:
do we need to terminate all user and common binary, if fault is in one of user app?

Reply:
Yes, all user binaries and common binary should be reloaded because they use same memory spaces.